src/o365/spo/commands/site/site-add.ts

```diff
@@ -356,10 +731,61 @@ class SpoSiteAddCommand extends SpoCommand {
     Create communication site using the Blank design with description and classification
       ${commands.SITE_ADD} --type CommunicationSite --url https://contoso.sharepoint.com/sites/marketing --title Marketing --description Site of the marketing department --classification MBI --siteDesign Blank
 
+    Create new classic site collection using the Team site template. Set time
+    zone to UTC+01:00. Don't wait for the site provisioning to complete
+      ${commands.SITE_ADD} --type ClassicSite --url https://contoso.sharepoint.com/sites/team --title Team --owner admin@contoso.onmicrosoft.com --timeZone 4
+
+    Create new classic site collection using the Team site template. Set time
+    zone to UTC+01:00. Wait for the site provisioning to complete
+      ${commands.SITE_ADD} --type ClassicSite --url https://contoso.sharepoint.com/sites/team --title Team --owner admin@contoso.onmicrosoft.com --timeZone 4 --webTemplate STS#0 --wait
+
+    Create new classic site collection using the Team site template. Set time
+    zone to UTC+01:00. If a site with the same URL is in the recycle bin, delete
+    it. Wait for the site provisioning to complete
+      ${commands.SITE_ADD} --type ClassicSite --url https://contoso.sharepoint.com/sites/team --title Team --owner admin@contoso.onmicrosoft.com --timeZone 4 --webTemplate STS#0 --removeDeletedSite --wait
+  
   More information
 
     Creating SharePoint Communication Site using REST
       https://docs.microsoft.com/en-us/sharepoint/dev/apis/communication-site-creation-rest
+
+  Remarks for classic sites:
```

Comment:
Remarks should go above the `More information` section to keep the docs consistent